addons/hr_holidays/static/src/components/thread_view/thread_view.js

```diff
@@ -2,7 +2,7 @@ odoo.define('hr_holidays/static/src/components/thread_view/thread_view.js', func
 'use strict';
 
 const components = {
-    ThreadView: require('mail/static/src/components/thread_view/thread_view.js'),
+    ThreadView: require('@mail/components/thread_view/thread_view')[Symbol.for("default")],
```

Comment:
Not for this PR specifically, but I think we should quickly make a pass and convert to new modules all the files that make use of `Symbol.for("default")`.

Reply:
Task: https://www.odoo.com/web#id=2510656&action=333&active_id=1519&model=project.task&view_type=form&cids=1&menu_id=4720